en/DatabaseProperties.md

```diff
@@ -1,28 +1,60 @@
 ---
 title: Database properties window
-outdated: true
 ---
 
 # Database properties window
 
-*Opened from the main window by selecting **File -&gt; Database properties***
+Each database can have specific properties that can be modified through **File -&gt; Database Properties**).
+These specific properties override the default properties (which are defined in **Options -&gt; Preferences**).
+
+The database-specific properties are stored in the database itself.
+This way, when moving the database to another computer, these properties are preserved. 
+In most of cases, special fields starting with *@Comment{jabref-meta:* are used. 
```

Comment:
Strictly these aren't special fields.

"In most of cases these are stored in the bib-file(database?) using text blocks(?) starting with *@Comment{jabref-meta:* ."? 

(I'm a bit reluctant to use "comment" here. But maybe it is better to just say "stored in comments" or something like that.)